scripts/collapse_replicates.R

```diff
@@ -1,29 +1,38 @@
+options(cli.unicode = FALSE)
+library(argparse)
+library(magrittr)
+library(tidyverse)
 suppressPackageStartupMessages(library(argparse))
 suppressPackageStartupMessages(library(magrittr))
-library(prismSeqR)
+source("/workspace/scripts/src/collapse_bio_reps.R")
```

Comment:
can this just be 
source("./src/collapse_bio_reps.R") ?

Reply:
I prefer to be explicit with our sourcing so things don't depend on the working dir. Because this is where podman clones the repo, these paths will always be correct full paths.